content/en/examples/pods/storage/projected-clustertrustbundle.yaml

```diff
@@ -4,12 +4,14 @@ metadata:
   name: sa-ctb-name-test
 spec:
   containers:
-  - name: container-test
+  - command:
+    - sleep
+    - "3600"
     image: busybox
-    command: ["sleep", "3600"]
+    name: container-test
```

Comment:
@eumel8 , looks fine to me because in general PodSpec  image and name key are always toghether.

Reply:
I agree with @utkarsh-singh1, this should stay the same.